### DISCUSSION TITLE
EGA 160x100 without black bars

### DISCUSSION THREAD

**Discussion by FrenkelS** [viti95/FastDoom#63] [Ideas]
I've read you've added an EGA 160x100 mode that's "Not 100% perfect as there are black bars up and down (80x350 resolution, 3 pixel tall characters)"

I haven't checked it out myself, but it seems [Paku Paku found a way to do EGA 160x100 without black bars](https://deathshadow.com/pakuPaku_Video).

**Comment by viti95** [comment]
Thanks for the idea! I've tested it, and works very well, but the brown color becomes a one close to pink (tested on both emulation and real hardware). I've tried to fix this problem but to no avail, and looks a little bit worse since the brown color is very used on 16 color modes. So for now I won't add support for it, until I can fix that brown color.

**Comment by FavoritoHJS** [comment]
Does that "close to pink" color look like intensified red? If so, the problem is that you programmed the pallete expecting it to output as 6-bit RGB, but it's being output as RGBI.

Reprogram the pallete as any other 200-line mode and all should be well.
